benchmarks/dbscan/print_timers.cpp

```diff
@@ -21,8 +21,12 @@ struct Timer
 {
   using clock_type = std::chrono::high_resolution_clock;
   std::string label;
-  clock_type::time_point tick;
-  clock_type::duration duration;
+  clock_type::time_point tick = {};
+  clock_type::duration duration = {};
+
+  Timer(std::string label)
+      : label(std::move(label))
+  {}
```

Comment:
Why not
```suggestion
  Timer(std::string const& label)
      : label(label)
  {}
```
If the label is in a variable prior to the call, would that not invalidate that variable?

Reply:
> If the label is in a variable prior to the call, would that not invalidate that variable?

No, we copy the label and then move it to the member variable, also see https://stackoverflow.com/a/51706522. In my opinion, it's more about style here, though. The performance consideration a rather minor anyway even if this was a performance critical function.